src/styles/blocks/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  text-transform: uppercase;
+
+  // box-shadow: 0 2px rgba(0, 0, 0, 0.05);
+
+  &__link-logo {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    position: relative;
+    display: flex;
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin-right: 20px;
+    height: 60px;
+
+    color: #{$main-accent};
+    text-decoration: none;
+  }
+
+  &__link--last-child {
+    margin-right: 0;
+  }
+
+  &__link:hover {
+    color: #00acdc;
```

Comment:
You defined this color #00acdc as #{$blue-accent} in variables. I recommend using it everywhere.